pkg/providers/cf/test_data/.cf/config.json

```diff
@@ -0,0 +1,43 @@
+{
+  "AccessToken": "bearer EXAMPLEEXAMPLERwczovL3VhYS5ib3NoLWxpdGUuY29tL3Rva2VuX2tleXMiLCJraWQiOiJrZXktMSIsInR5cCI6IkpXVCIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJjYWU0YTNhZC0yNDhiLTRmYzctOWFlNS02NDI0YWY4NWMyZjkiLCJ1c2VyX25hbWUiOiJhZG1pbiIsIm9yaWdpbiI6InVhYSIsImlzcyI6Imh0dHBzOi8vdWFhLmJvc2gtbGl0ZS5jb20vb2F1dGgvdG9rZW4iLCJjbGllbnRfaWQiOiJjZiIsImF1ZCI6WyJzY2ltIiwiY2xvdWRfY29udHJvbGxlciIsInBhc3N3b3JkIiwiY2YiLCJjbGllbnRzIiwidWFhIiwib3BlbmlkIiwiZG9wcGxlciIsInJvdXRpbmcucm91dGVyX2dyb3VwcyIsIm5ldHdvcmsiXSwiemlkIjoidWFhIiwiZ3JhbnRfdHlwZSI6InBhc3N3b3JkIiwidXNlcl9pZCI6ImNhZTRhM2FkLTI0OGItNGZjNy05YWU1LTY0MjRhZjg1YzJmOSIsImF6cCI6ImNmIiwic2NvcGUiOlsiY2xpZW50cy5yZWFkIiwib3BlbmlkIiwicm91dGluZy5yb3V0ZXJfZ3JvdXBzLndyaXRlIiwic2NpbS5yZWFkIiwiY2xvdWRfY29udHJvbGxlci5hZG1pbiIsInVhYS51c2VyIiwicm91dGluZy5yb3V0ZXJfZ3JvdXBzLnJlYWQiLCJjbG91ZF9jb250cm9sbGVyLnJlYWQiLCJwYXNzd29yZC53cml0ZSIsImNsb3VkX2NvbnRyb2xsZXIud3JpdGUiLCJuZXR3b3JrLmFkbWluIiwiZG9wcGxlci5maXJlaG9zZSIsInNjaW0ud3JpdGUiXSwiYXV0aF90aW1lIjoxNzQ3MzkwNjAwLCJleHAiOjE3NDczOTE4MDAsImlhdCI6MTc0NzM5MDYwMCwianRpIjoiOTU4NDQwNWJiOTU1NDYyMGE4Yzg5MDNkZWRhNzlkOGMiLCJlbWFpbCI6ImFkbWluIiwicmV2X3NpZyI6IjJmMTU5YmEiLCJjbGllbnRfYXV0aF9tZXRob2QiOiJub25lIiwiY2lkIjoiY2YifQ.bRDsTqEmRX6YEvLVst7HIovjjj9owb9IHNu9CH_kEFUvdg6_KMeg4_GmOAhgF_DQLmuZg1jz565_KK4C9aIZt3RL_V0lyWjWa7brrko380CN1_UVnjsvu3u3eQQEXAMPLEAeNRQ1Xj63pTp6fkYc67I1_Q4_qvMHkbCnmOFF-VtGTjdaON-5TA8JOrNVmCdekkVqDgLLNeX15Pyopjdg_mHB1-s3mKYLSXOafEi8Ah55diDJOzSgYsSujeEZvO9mQT6whuh9mFMZ9TNTmUtXADbUVTY35FIX-TmJDDZx_483OZt06Q5XyhnRD6XmmKk3Y0zWBS1NEXAMPLEEXAMPLE",
```

Comment:
does the bearer token need to be this long?

Reply:
yes, tried to use a shorter one but it was invalid. 
I added `EXAMPLE` chars at the beginning and the end of the token to clarify it's a fake one.